### PR TITLE
Exposing the structured GCSE fields on the V2 API

### PR DESCRIPTION
**Pull request by george-bit**
### Context

Carrying on from the attributes added on #1977  we need to expose these attributes on the V2 TTAPI for use in Publish.

### Changes proposed in this pull request

This PR exposes the attributes and adds specs to cover this behaviour.

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: spec/factories/courses.rb [hunk 1]
```diff
@@ -26,6 +26,13 @@
     additional_degree_subject_requirements { true }
     degree_subject_requirements { Faker::Lorem.sentence.to_s }
 
+    accept_pending_gcse { true }
+    accept_gcse_equivalency { true }
+    accept_english_gcse_equivalency { true }
+    accept_maths_gcse_equivalency { false }
+    accept_science_gcse_equivalency { false }
```
Comment (defong) [inline spec/factories/courses.rb:33]: Is there a particular reaon if not can we user `[true, false].sample`

**Review by defong — CHANGES_REQUESTED** [review]
Failing test? Otherwise I think its fine

File: spec/factories/courses.rb (hunk 1 above)
Comment (george-bit) [reply spec/factories/courses.rb:33]: My bad, added the factory in last so didn't check to see if it broke anything. Will fix as suggested 👍

Comment (george-bit) [reply spec/factories/courses.rb]: okay so had a bit of trouble testing the output of  `[true, false].sample` in the stubbed API response. Therefore, I've packaged these attributes as a trait on the courses factory. Is this okay?

**Review by defong — APPROVED** [review]

**Pull request merged**